### DISCUSSION TITLE
cfclient not working

### DISCUSSION THREAD

**Discussion by nrlhozkan** [bitcraze/discussions#751] [Q&A] [answered]
I was using an older version of cfclient. I removed and installed it again now it is not working. 

The error : INFO:cfclient.gui:Disabling STL printouts
INFO:cfclient.utils.input.inputreaders:Input readers: ['linuxjsdev', 'pysdl2']
INFO:cfclient.utils.input.inputreaders:Successfully initialized [linuxjsdev]
INFO:cfclient.utils.input.inputreaders:Could not initialize [pysdl2]: No SDL2 support on Linux
INFO:cfclient.utils.input.inputinterfaces:Found interfaces: ['leapmotion', 'wiimote', 'zmqpull']
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [leapmotion]: Leap Motion library probably not installed (No module named 'leapsdk')
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [wiimote]: Missing cwiid (wiimote) driver No module named 'cwiid'
INFO:cfclient.utils.config:Dist config read from /home/ozkan/crazyflie-clients-python/src/cfclient/configs/config.json
INFO:cfclient.utils.config:Config file read from [/home/ozkan/.config/cfclient/config.json]
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [zmqpull]: ZMQ input disabled in config file
INFO:OpenGL.acceleratesupport:OpenGL_accelerate module loaded
INFO:OpenGL.arrays.arraydatatype:Using accelerated ArrayDatatype
INFO:cfclient.utils.zmq_led_driver:ZMQ led disabled in config file
INFO:cfclient.utils.zmq_param:ZMQ param disabled in config file
INFO:cfclient.gui:If startup fails because of "xcb", install dependency with `sudo apt install libxcb-xinerama0`.
QObject::moveToThread: Current thread (0x2593ad0) is not the object's thread (0x42ca500).
Cannot move to target thread (0x2593ad0)

qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "/home/ozkan/.local/lib/python3.8/site-packages/cv2/qt/plugins" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: xcb, eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, wayland-egl, wayland, wayland-xcomposite-egl, wayland-xcomposite-glx, webgl.

Aborted (core dumped)

I could not figure out.

**Comment by krichardsson** [comment]
Which version are you trying to use?

**Reply by nrlhozkan** [reply]
Thank you for the replay. I want to use the new version.

**Comment by nrlhozkan** [comment, marked as answer]
https://github.com/ageitgey/face_recognition/issues/1041#issuecomment-582401795 

Using the above link helps me to solve the problem. Thanks
